docs/modules/mssqlfg.md

```diff
@@ -0,0 +1,300 @@
+
+
+Additional Provision Parameters for : premium plan
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| `dtus` | `integer` | Specifies Database transaction units, which represent a bundled measure of compute, storage, and IO resources. Valid values are 125, 250, 500, 1000, 1750, 1000 | N | 125 |
+
+Additional Provision Parameters for: general-purpose
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| `cores` | `integer` | Specifies vCores, which represent the logical CPU. Valid values are 2, 4, 8, 16, or 24, 32, 48, 80 | N | 2 |
+| `storage` | `integer` | Specifies the amount of storage to allocate in GB. Ranges from 5 to 1048 | N | 5 |
+
+Additional Provision Parameters for: business-critical
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| `cores` | `integer` | Specifies vCores, which represent the logical CPU. Valid values are 2, 4, 8, 16, or 24, 32, 48, 80 | N | 2 |
+| `storage` | `integer` | Specifies the amount of storage to allocate in GB. Ranges from 5 to 1048 | N | 5 |
+
+##### Bind
+
+Creates a new user on the primary SQL Database. (The secondary database syncs the creation.) The new user will be named randomly and granted permission to log into and administer the database.
+
+###### Binding Parameters
+
+This binding operation does not support any parameters.
+
+###### Credentials
+
+Binding returns the following connection details and credentials:
+
+| Field Name | Type | Description |
+|------------|------|-------------|
+| `host` | `string` | The fully-qualified address of the Failover Group. |
+| `port` | `int` | The port number to connect to on the SQL Server. |
+| `database` | `string` | The name of the database. |
+| `username` | `string` | The name of the database user. |
+| `password` | `string` | The password for the database user. |
+| `uri` | `string` | A uri string containing connection information. |
+| `jdbcUrl` | `string` | A fully formed JDBC url. |
+| `encrypt` | `boolean` | Flag indicating if the connection should be encrypted. |
+| `tags` | `string[]` | List of tags. |
+
+##### Update
+
+Updates both the primary database and the secondary database.
+
+###### Updating Parameters
+
+Parameters for updating the SQL DB Database differ by plan. See each section for relevant parameters.
+
+Additional Provision Parameters for : standard plan
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| `dtu` | `integer` | Specifies Database transaction units, which represent a bundled measure of compute, storage, and IO resources. Valid values are 10, 20, 50, 100, 200, 400, 800, 1600, 3000 | N | 10 |
```

Comment:
Why the parameter name is "dtus" in provision, but here the name is "dtu"? Is that by design?

Reply:
I copied it from mssql. Thanks for catching the typo...Will fix.